book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ]  Hello there! 🤗 Please make sure to include a setter for the @rental variable. This is important because we'll need to not only read but also make modifications to it. 📝✏️ By adding a setter, we can ensure that the @rental variable becomes more flexible and adaptable, allowing us to make the necessary changes as required. 🔧💼

```suggestion
 attr_accessor :title, :author, :rentals
```